wordpressorg.dev/provision/wp-config.php

```diff
@@ -83,6 +82,13 @@
 	define( 'SITE_ID_CURRENT_SITE', 1 );
 	define( 'BLOG_ID_CURRENT_SITE', 1 );
 	define( 'SUBDOMAIN_INSTALL',    false );
+	
+} elseif ( $_SERVER['HTTP_HOST'] === 'translate.wordpressorg.dev' ) {
+	define( 'GLOTPRESS_TABLE_PREFIX', 'translate_' );
```

Comment:
`GLOTPRESS_TABLE_PREFIX` is used by other sites too. Let's add it here https://github.com/WordPress/meta-environment/blob/master/wordpressorg.dev/provision/wp-config.php#L58